examples/music_drop_fade.py

```diff
@@ -122,12 +122,12 @@ def draw_text_line(text, y=0):
     return y
 
 
-def change_music_postion(amount):
+def change_music_position(amount):
```

Comment:
This function name fix does not have the needed corresponding fixes in function calls in this file

Reply:
Ah, thank you!